### PR TITLE
Add page on how to cite project or packages

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
Closes https://github.com/scikit-hep/scikit-hep.github.io/issues/43.

### CODE REVIEW

File: pages/about/citing.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+layout: page
+title: Citing
+permalink: /citing
+nav_order: 6
+parent: About
```
Comment (HDembinski) [inline pages/about/citing.md:5]: I think it would be better to have this as a top-level entry, just to make sure it is clearly visible.

Comment (HDembinski) [reply pages/about/citing.md:5]: If it is under about, you need to click on about to unfold it and see it.

Comment (eduardo-rodrigues) [reply pages/about/citing.md]: OK, fair enough. I have no strong opinion. Moved to the top level.

**Comment by eduardo-rodrigues** [comment]
Thanks for the approval, Hans. Let's wait just a bit more for others to comment. I will then merge by the end of the day or so.

File: pages/citing.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    journal = "EPJ Web Conf.",
+    volume = "245",
+    pages = "06028",
+    year = "2020"
+}
+```
+
+Kindly consider citing any specific package if that seems appropriate.
+All package GitHub repositories contain at the very least a Zenodo badge.
+Some may also have a citation page, as for example https://github.com/scikit-hep/iminuit/blob/develop/CITATION.rst.
```
Comment (henryiii) [inline pages/citing.md:32]:
```suggestion
Some may also have a citation page, as for example <https://github.com/scikit-hep/iminuit/blob/develop/CITATION.rst>.
```

File: pages/citing.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+---
+
+
+Citing Scikit-HEP or individual packages {#citing}
+========================================
+
+We appreciate if you consider to acknowledge the Scikit-HEP project in case it has made or is having an impact on your work or research.
+We suggest the following citation for the project as a whole (in BibTeX format):
+
+```
```
Comment (henryiii) [inline pages/citing.md:14]:
```suggestion
```BibTeX
```

**Review by henryiii — APPROVED** [review]

File: pages/citing.md (hunk 3 above)
Comment (eduardo-rodrigues) [reply pages/citing.md]: Thanks Henry. Funny enough I was looking for something like that but did not find it from a quick search.

**Pull request merged**